### PR TITLE
Development

### PR DESCRIPTION
**Pull request by nitsandaniel1**
Added an alert when the folder is full

### CODE REVIEW

**Comment by nitsandaniel1** [comment]
also added env_requirements.txt

File: env_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
+https://repo.anaconda.com/pkgs/main/win-64/git-2.23.0-h6bb4b03_0.conda
```
Comment (theitush) [inline env_requirements.txt]: how would you install the requirements using this file?
could you add instructions to the README ?

File: runner.py [hunk 2]
```diff
@@ -279,6 +279,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
            quality_threshold, task, evalue, dust, num_alignments, soft_masking, perc_identity, mode, max_read_size,
            with_indels, stretches_pvalue, stretches_distance, stretches_to_plot, cleanup,
            cpu_count, overlapping_reads, db_path, max_memory, calculate_haplotypes="Y"):
+    if os.path.exists(output_dir): # the output folder must be empty
```
Comment (theitush) [inline runner.py]: note bellow that output_dir can be None which would result in an error here...
So this check should happen after assigning output_dir (a few lines down)

Comment (theitush) [reply runner.py]: also the comment is redundant

File: runner.py [hunk 3]
```diff
@@ -279,6 +279,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
...
+        if os.listdir(output_dir):
+            raise Exception("The output folder is not empty. Empty it or choose a different name.")
```
Comment (theitush) [inline runner.py]: should stick to existing variable names and be more concise so something like: "output_dir must be a path to a new or empty directory"

**Review by theitush — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,2 @@
-This is a work in progress of a python implementation of the entire pipeline.
\ No newline at end of file
+This is a work in progress of a python implementation of the entire pipeline.
+In order to install the environment: conda create --file requirements.txt name_of_new_environment
```
Comment (theitush) [inline README.md]: great.
Only that it's env_requirements.txt.. please be consistent..!

File: runner.py [hunk 5]
```diff
@@ -283,6 +283,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
         db_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'db')
     if not output_dir:
         output_dir = assign_output_dir(db_path)
+    if os.path.exists(output_dir): # the output folder must be empty
```
Comment (theitush) [inline runner.py]: redundant comment

File: runner.py [hunk 6]
```diff
@@ -283,6 +283,9 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
...
+        if os.listdir(output_dir):
+            raise Exception("The output folder is not empty. Empty it or choose a different name.")
```
Comment (theitush) [inline runner.py]: please maintain consistency in variable names - "output folder" should be "output_dir".
text is also not clear enough.
Recommend changing to something like: "output_dir must be path to a new or empty directory!"

File: env_requirements.txt (hunk 1 above)
Comment (theitush) [inline env_requirements.txt]: This file should be removed.

**Review by theitush — APPROVED** [review]
Mazal Tov!

**Pull request merged**